tdrs-frontend/reports/zap.conf

```diff
@@ -0,0 +1,10 @@
+90034	IGNORE	(Cloud Metadata potentially exposed)
+90022	IGNORE	(Application Error Disclosure)
+40034	IGNORE	(.env information leak)
```

Comment:
The env leak was showing up before we added the 404 pages probably because all pages appeared to be successful, a few of these were like this. I will check which problems are still happening after the 404 pages were added, and update this list accordingly

Reply:
The only two that are currently showing up are:
```
IGNORE-NEW: Server Leaks Version Information via "Server" HTTP Response Header Field [10036] x 2 
	http://web:8080/ (404 Not Found)
	http://web:8080/sitemap.xml (404 Not Found)
IGNORE-NEW: Content Security Policy (CSP) Header Not Set [10038] x 2 
	http://web:8080/ (404 Not Found)
	http://web:8080/sitemap.xml (404 Not Found)
FAIL-NEW: 0	FAIL-INPROG: 0	WARN-NEW: 0	WARN-INPROG: 0	INFO: 0	IGNORE: 2	PASS: 113
```